katello/rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec

```diff
@@ -0,0 +1,127 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name hammer_cli_csv
+%global confdir hammer
+
+Summary: CSV input/output command plugin for the Hammer CLI
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.3.0
+Release: 1%{?dist}
+Group:   Development/Languages
+License: GPLv3
+URL:     https://github.com/Katello/hammer-cli-csv
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 1.0.1-7
+%endif
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli_katello)
+
+BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+
+%description
+CSV input/output command plugin for the Hammer CLI.
+
+%package doc
+Summary:   Documentation for %{pkg_name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-doc}
+
+%description doc
+Documentation for %{pkg_name}
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - <<EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d
+install -m 755 .%{gem_instdir}/config/csv.yml %{buildroot}%{_root_sysconfdir}/%{confdir}/cli.modules.d/csv.yml
```

Comment:
This can be 644 I think

Reply:
From what I can tell, this is true in all hammer packages. I'd prefer not to one off change a few and make this change across the board.